repo/config/init.go

```diff
@@ -120,7 +120,6 @@ func Init(version string) (*Config, error) {
 		},
 		Logs: Logs{
 			LogToDisk: true,
-			LogLevel:  "error",
```

Comment:
shouldn't need to worry about existing configs by removing this i think?

Reply:
perfect